<launch>
  <node pkg="chairbot_selfdrive" type="pose2frame" args="pose:=pf_filtered" name="pose2frame_3d"/>
  <node pkg="chairbot_selfdrive" type="pose2frame" args="pose:=pf_filtered _mode_2d:=true" name="pose2frame_2d"/>
</launch>
